runtime/package.json

```diff
@@ -17,7 +17,7 @@
     "clean:dist": "npm run rimraf -- dist",
     "clean:install": "npm set progress=false && npm install",
     "clean": "npm --force cache clean && npm run rimraf -- node_modules doc coverage dist compiled dll",
-    "reinstall": "rimraf node_modules && npm --force cache clean && npm install",
+    "reinstall": "rm -rf node_modules && npm --force cache clean && npm install",
```

Comment:
Finally someone did it.
Although, with this, people on Windows can't use the `reinstall` script anymore (is there anyone who uses Windows?).

Reply:
I don't think anyone uses windows. ;)